From 20537f77e4b66c5e657174a4f29dfaf883b8929c Mon Sep 17 00:00:00 2001
From: Pablo Ceballos <pceballos@google.com>
Date: Fri, 13 May 2022 23:33:07 +0000
Subject: [PATCH] hwdb: Add Google Hangouts Meet speakermic

This device implements the phone mute HID usage as a toggle switch,
where 1 indicates muted, and 0 indicates unmuted. However, for a key
event 1 indicates that the key has been pressed and 0 indicates it has
been released. This mismatch causes issues, so prevent key events from
being generated for this HID usage.

(cherry picked from commit 2d251543211dae43bcf6f01ed32a3dbe8fef1ea7)

Related: #2087778
---
 hwdb.d/60-keyboard.hwdb | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/hwdb.d/60-keyboard.hwdb b/hwdb.d/60-keyboard.hwdb
index b852f33608..2e4f6f4293 100644
--- a/hwdb.d/60-keyboard.hwdb
+++ b/hwdb.d/60-keyboard.hwdb
@@ -511,6 +511,14 @@ evdev:input:b0003v0458p0708*
  KEYBOARD_KEY_0900fd=scale
  KEYBOARD_KEY_0900fc=screenlock
 
+###########################################################
+# Google
+###########################################################
+
+# Google Hangouts Meet speakermic
+evdev:input:b0003v18D1p8001*
+ KEYBOARD_KEY_b002f=reserved                            # Disable micmute key
+
 ###########################################################
 # Hewlett Packard
 ###########################################################
